akka-docs/src/main/paradox/stream/operators/Source/sinkToSource.md

```diff
@@ -0,0 +1,19 @@
+# Source.sinkToSource
+
+Materializes into a sink connected to a source.
+
+@ref[Source stages](../index.md#source-stages)
+
+@@@div { .group-scala }
+
+## Signature
+
+@@signature [Source.scala]($akka$/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #sinkToSource }
+
+@@@
+
+## Description
+
+@@@div { .callout }
+
+@@@
```

Comment:
we'd need examples as well, from nowadays we should always right away include examples when we merge new operators. See here how: https://raw.githubusercontent.com/akka/akka/master/akka-docs/src/main/paradox/stream/operators/Sink/actorRefWithAck.md